pkg/clients/clients.go

```diff
@@ -44,32 +43,23 @@ func init() {
 }
 
 // GetKubeClientSetOrDie Add a native non-caching client for advanced CRUD operations
-func GetKubeClientSetOrDie() kubernetes.Clientset {
-
-	clientSet, err := kubernetes.NewForConfig(RestConfig)
-	exit.OnError(err)
-	return *clientSet
+func GetKubeClientSetOrDie() (*kubernetes.Clientset, error) {
+	return kubernetes.NewForConfig(RestConfig)
```

Comment:
This changes the behaviour of the function. It's not dying if it can't get the ClientSet anymore.